tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -68,6 +68,7 @@ platform_tests/api/test_chassis.py::TestChassisApi::test_get_watchdog:
     conditions:
       - "asic_type in ['barefoot'] and hwsku in ['newport']"
       - "'sw_to3200k' in hwsku"
+      - "'Force10-S6000' in hwsku"
```

Comment:
@assrinivasan day-1 issue? 6000 never had watchdog support from 201811?

Reply:
That's correct Prince: https://github.com/sonic-net/sonic-buildimage/tree/201811/platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform